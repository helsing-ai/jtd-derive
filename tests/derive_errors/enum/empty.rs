#[derive(jtd_derive::JsonTypedef)]
enum Empty {}

fn main() {}
