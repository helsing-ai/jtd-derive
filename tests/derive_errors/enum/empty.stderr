error: enums with no variants are unsupported
 --> tests/derive_errors/enum/empty.rs:1:10
  |
1 | #[derive(jtd_derive::JsonTypedef)]
  |          ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `jtd_derive::JsonTypedef` (in Nightly builds, run with -Z macro-backtrace for more info)
